genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -135,9 +135,13 @@ to work.
 |false
 
 |genie.jobs.users.activeLimit.count
-|The maximum number of active jobs a user is allowed to have. Once a user hits this limit, jobs submitted are rejected. This is property is ignored unless `genie.jobs.users.activeLimit.enabled` is set to true.
+|The maximum number of active jobs a user is allowed to have. Once a user hits this limit, jobs submitted are rejected. This is property is ignored unless `genie.jobs.users.activeLimit.enabled` is set to true. This limit applies to users that don't have a more specific value set, see below.
```

Comment:
Rather than `see below` maybe a direct reference to `see genie.jobs.users.activeLimit.specialUser.user-name` cause below could change at any time if someone copies this without knowing?